### ISSUE TITLE
[Request] gnome-shell-extension-blur-my-shell

### ISSUE THREAD

**Issue by abraham-sah** [chaotic-aur/packages#2389]
### Link to the package base(s) in the AUR

https://aur.archlinux.org/packages/gnome-shell-extension-blur-my-shell  
https://aur.archlinux.org/packages/gnome-shell-extension-blur-my-shell-git

### Utility this package has for you

Extension that adds a blur look to different parts of the GNOME Shell

### Do you consider the package(s) to be useful for every Chaotic-AUR user?

No, but for a few.

### Do you consider the package to be useful for feature testing/preview?

- [ ] Yes

### Have you tested if the package builds in a clean chroot?

- [ ] Yes

### Does the package's license allow redistributing it?

YES!

### Have you searched the issues to ensure this request is unique?

- [X] YES!

### Have you read the README to ensure this package is not banned?

- [X] YES!

### More information

It's the #12 most downloaded Gnome extension between Gnome users of all distros with about "1089452 downloads". And, natively installed in some distros, even Clear-Linux.

**Comment by Technetium1** [comment]
Is there a benefit to also adding `gnome-shell-extension-blur-my-shell-git` alongside stable?

**Comment by abraham-sah** [comment]
> Is there a benefit to also adding gnome-shell-extension-blur-my-shell-git alongside stable?

No, I don't think so... Nevertheless, I provided the link of git AUR page, just for your information.
